cards/en/cel25c.json

```diff
@@ -653,7 +664,7 @@
       "Psychic"
     ],
     "rules": [
-      "When a Pokémon-ex has been knocked out, your opponent takes 2 Prize cards."
+      "When Pokémon-ex has been knocked out, your opponent takes 2 Prize cards."
```

Comment:
I think this should be this: `Pokémon-EX rule: When a Pokémon-EX has been Knocked Out, your opponent takes 2 Prize cards.`